.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on:
+    push:
+    pull_request:
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go 1.13
```

Comment:
The go version moves, right? So you should just add Set up go and remove the "1.13"

Reply:
Corrected.